app/components/markdown/index.js

```diff
@@ -3,15 +3,17 @@
 
 import {Parser} from 'commonmark';
 import Renderer from 'commonmark-react-renderer';
-import React, {PureComponent} from 'react';
+import React from 'react';
```

Comment:
I changed it to be consistent with the web app since we went back to using `React.PureComponent` there